Lombiq.Tests.UI/Lombiq.Tests.UI.csproj

```diff
@@ -92,6 +93,7 @@
     <PackageReference Include="Lombiq.Tests" Version="2.2.0" />
     <PackageReference Include="Lombiq.HelpfulLibraries.Cli" Version="5.1.1" />
     <PackageReference Include="Lombiq.HelpfulLibraries.OrchardCore" Version="5.1.1" />
+    <PackageReference Include="Lombiq.HelpfulLibraries.Refit" Version="5.1.1" />
```

Comment:
This doesn't exist and will fail the next NuGet publish. Publish an issue-specific alpha for it, please. Same for the API client project.